cg/services/order_validation_service/workflows/tomte/models/sample.py

```diff
@@ -20,6 +20,6 @@ class TomteSample(Sample):
     sex: SexEnum | None = None
     source: str | None = None
     status: StatusEnum | None = None
-    subject_id: str = Field(pattern=NAME_PATTERN, max_length=128)
+    subject_id: str = Field(pattern=NAME_PATTERN, min_length=1, max_length=128)
```

Comment:
Is this correct if an existing sample is used?

Reply:
Remains to be seen! Might change in a future PR